docs/pages/setup/reference/cli.mdx

```diff
@@ -1310,6 +1310,346 @@ $ tctl auth rotate --type=user --grace-period=200h
+
+```bsh
+$ tctl get saml/your-connector-name --with-secrets | tctl sso test
+````
+
+<Admonition
+  type="note"
+  title="Important"
+>
+  Make sure to include `--with-secrets` flag, or the exported auth connector will not be testable.
+</Admonition>
+
+### tctl sso configure github
+
+Configure GitHub auth connector.
+
+Required params `--id` and `--secret` come from the [Github OAuth app](https://docs.github.com/en/developers/apps/building-oauth-apps/creating-an-oauth-app).
+
+The flag `--teams-to-roles` can be provided multiple times to specify which Github Teams are assigned which roles.
+In this example, members of the `octocats` team in GitHub will be assigned the `devs`, `access`, and `editor` roles in Teleport.
+
+```code
+$ tctl sso test --id=GITHUB_CLIENT_ID --secret=GITHUB_SECRET --teams-to-roles=octocats,devs,access,editor [<other-flags>]
+```
+
+#### Arguments
+
+This command accepts no arguments.
+
+#### Flags
+
+Mandatory flags: `--id`, `--secret`, `--teams-to-roles`.
+
+| Name                      | Default Value(s) | Allowed Value(s)           | Description                                             |
+|---------------------------|------------------|----------------------------|---------------------------------------------------------|
+| `-n`, `--name`            | `github`         | resource name              | Connector name.                                         |
+| `-r`, `--teams-to-roles`  | none             | `org,team,role1,role2,...` | Sets teams-to-roles mapping. Repeatable.                |
+| `--display`               | none             | display name               | Controls how this connector is displayed.               |
+| `--id`                    | none             | Github OAuth2 client id    | Github app client ID.                                   |
+| `--secret`                | none             | Github OAuth2 secret       | Github app client secret.                               |
+| `--redirect-url`          | none             | Valid callback URL.        | Authorization callback URL.                             |
+| `--ignore-missing-roles`  |                  |                            | Ignore missing roles referenced in `--teams-to-roles`.  |
+
+#### Global flags
+
+These flags are available for all commands `--debug, --config`. Run
+`tctl help <subcommand>` or see the [Global Flags section](#tctl-global-flags).
+
+#### Examples
+
+Generate GitHub auth connector. Two role mappings are defined:
+  - members of `admin` team in `octocats` org will receive `access`, `editor` and `auditor` roles.
+  - members of `dev` team in `octocats` org will receive `access` role.
+
+The values for `--secret` and `--id` are provided by GitHub on [OAuth app](https://docs.github.com/en/developers/apps/building-oauth-apps/authorizing-oauth-apps) configuration page.
+
+```code
+$ tctl sso configure gh -r octocats,admin,access,editor,auditor -r octocats,dev,access --secret GH_SECRET --id CLIENT_ID
```

Comment:
Should we specify earlier that `gh` is an alias for `github`?

Reply:
Added.